Makefile

```diff
@@ -113,6 +113,11 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
 
+.PHONY: render-manifest
+render-manifest: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
```

Comment:
Very similar target in kyma/components/function-controller is named "manifests". Is this new way of naming or could we standarize these names?

Reply:
ok